azure-pipelines-20230430.yml

```diff
@@ -61,7 +61,7 @@ parameters:
     default:
       - 'hudi-spark-datasource'
       - 'hudi-spark-datasource/hudi-spark'
-      - 'hudi-spark-datasource/hudi-spark3.2.x'
+      - 'hudi-spark-datasource/hudi-spark3.5.x'
```

Comment:
Keep Azure CI the same as there are test failures that need to be addressed in #11493.